substrate/frame/contracts/src/wasm/mod.rs

```diff
@@ -3046,6 +3068,337 @@ mod tests {
 		assert_eq!(&result.data[4..], &[0u8; 0]);
 	}
 
+	#[test]
```

Comment:
🤔  @athei at some point,  shouldn't we translate these code examples to Rust like we did with the other ones

Reply:
Yeah we can just remove those tests. It is just boilerplate and everything is testet in the fixtures anyways.